### DISCUSSION TITLE
reference to simple column example

### DISCUSSION THREAD

**Discussion by francispoulin** [OceanBioME/OceanBioME.jl#216] [General]
Is the physical model used in the `simple column example` from a paper or another resource?  If yes, I would very much like to know which one so that I can read more about where this comes from.

**Comment by johnryantaylor** [comment]
Hi @francispoulin, we used something similar to this in this paper: https://www.frontiersin.org/journals/marine-science/articles/10.3389/fmars.2024.1359614/full, but the details were different.

**Reply by francispoulin** [reply]
Thank you very much for the informative and quick reply!
